File: OCH_Tao/ABC/B/369.py
N = int(input())
L,R = 0,0
cnt = 0
for i in range(N):
  A,S = input().split()
  A = int(A)
  if S=="L":
    if L!=0:
      cnt+=abs(L-A)
    L = A
  else:
    if R!=0:
      cnt+=abs(R-A)
    R = A
print(cnt)

File: OCH_Tao/ABC/B/369.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
input.shift();
let [L, R, cnt] = [0, 0, 0];
for (const i of input) {
  let [A, S] = i.split(" ");
  A = parseInt(A, 10);
  if (S === "L") {
    if (L !== 0) {
      cnt += Math.abs(L - A);
    }
    L = A;
  } else {
    if (R !== 0) {
      cnt += Math.abs(R - A);
    }
    R = A;
  }
}
console.log(cnt);